pandas/core/ops/__init__.py

```diff
@@ -372,6 +373,11 @@ def dispatch_to_series(left, right, func, str_rep=None, axis=None):
     right = lib.item_from_zerodim(right)
     if lib.is_scalar(right) or np.ndim(right) == 0:
 
+        # Get the appropriate array-op to apply to each block's values.
+        array_op = get_array_op(func, str_rep=str_rep)
+        bm = left._data.apply(array_op, right=right)
+        return type(left)(bm)
+
```

Comment:
this could just be an if (as you are returning), e.g. change the following elif to an if, but NBD